### ISSUE TITLE
Accessibility: no way to set Label control accessible role and accessible name

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3294]
* .NET Core Version: 

Reproduces in 3.1 as well as in 5.0.

* Have you experienced this same bug with .NET Framework?: 

Yes
Link to DevDiv issue: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1108206

**Problem description:**

If AccessibleName or/and AccessibleRole is set to a custom value for the Label control, this does not affect the AccessibleName and AccessibleRole in runtime. And default values are shown.

**Expected behavior:**
There should be an ability to set/change AccessibleName and AccessibleRole for the Label control.

**Minimal repro:**
Create the app with the Label control . Specify custom AccessibleName and AccessibleRole for the label. Run the app and observe that the Default AccessibleRole and default AccessibleName is show in Inspect or Accessibility Insights.

**Comment by Tanya-Solyanik** [comment]
@vladimir-krestov - could you please create a new "servicing-consider" bug for 3.1 when the list of affected controls is finalized?

**Comment by vladimir-krestov** [comment]
Okay, I will.

**Comment by vladimir-krestov** [comment]
#3504
